### ISSUE TITLE
Callback is called twice on destroy when session does not exist

### ISSUE THREAD

**Issue by agubler** [kleydon/prisma-session-store#72]
It looks like the callback to destroy is called twice when the session does not exist, once with the error argument and then again for the call when the action finishes without an error.

```ts
  public readonly destroy = async (
    sid: string | string[],
    callback?: (err?: unknown) => void
  ) => {
    if (!(await this.validateConnection())) return callback?.();

    try {
      if (Array.isArray(sid)) {
        await Promise.all(sid.map(async (s) => this.destroy(s, callback)));
      } else {
        await this.prisma[this.sessionModelName].delete({ where: { sid } });
      }
    } catch (e: unknown) {
      this.logger.warn(
        `Attempt to destroy non-existent session:${String(sid)} ${String(e)}`
      );
      if (callback) defer(callback, e);
    }
    // This is called even if there is an error above
    if (callback) defer(callback);
  };
```

**Comment by kleydon** [comment]
@agubler - Good catch!
I'll put together a PR for this fix.

**Comment by kleydon** [comment]
Fixed in v3.1.3; thanks for the bug report.
